contrib/presets/icons/more-unicode.conf

```diff
@@ -0,0 +1,56 @@
+# This preset use fancier characters than the default,
+# but they should still be in most unicode fonts.
+
+LP_TIME_FORMAT="%Hₕ%Mₘ%Sₛ"
+LP_DELIMITER_KUBECONTEXT_PREFIX="「"
+LP_DELIMITER_KUBECONTEXT_SUFFIX="」"
+LP_MARK_ADAPTER="⏚"
+LP_MARK_BATTERY="⌁"
+LP_MARK_BRACKET_CLOSE="⦘"
+LP_MARK_BRACKET_OPEN="⦗"
+LP_MARK_BZR="⚯"
+LP_MARK_CMAKE="·"
+LP_MARK_DEV_CLOSE="❭"
+LP_MARK_DEV_MID="|"
+LP_MARK_DEV_OPEN="❬"
+LP_MARK_DIRSTACK="⚞"
+LP_MARK_DISABLED="⌀"
+LP_MARK_FOSSIL="⌘"
+LP_MARK_GIT="±"
```

Comment:
Aren't these the defaults already? I'd rather see this file contain only things different than the default.

Reply:
Good catch, I don't know how you can see that ^^